packages/@okta/vuepress-site/docs/reference/api/authn/index.md

```diff
@@ -871,19 +871,27 @@ Content-Type: application/json
 
 Include the `X-Device-Fingerprint` header to supply a device fingerprint. The device fingerprint is used in the following ways:
```

Comment:
to be consistent with product doc, "device fingerprint" should be "browser fingerprint". See https://oktawiki.atlassian.net/wiki/spaces/~damian.ali/pages/886806858/SIP+New+Device+Behavior for what's going into product help.

Reply:
I think we should continue to call the value for `X-Device-Fingerprint` header as a device fingerprint in the API docs. The documentation for browser flows in https://oktawiki.atlassian.net/wiki/spaces/~damian.ali/pages/886806858/SIP+New+Device+Behavior talks about the JS fingerprint that is generated by the Okta SIW, which can have FP and FN. For these trusted app API scenarios, the customer may be generating a UUID, storing in a secure cookie and passing it along in the header and in such cases, what they are sending is not the browser fingerprint. So, for the trusted app scenarios, I think it would be confusing if we call this browser fingerprint..

Reply:
Let's go with `X-device-Fingerprint` where it says "device fingerprint" here

Reply:
Let's go with X-Device-Fingerprint where it says "device fingerprint" here